src/posts/2021-05-03/index.md

```diff
@@ -0,0 +1,659 @@
+  practice. From the overview of the proposal, this happens after an increasing
+  number of users have requested a donation to a foundation, to get the final
+  piece of insurance on the commitment to open-source required for building
+  large projects on top of Cilium. The CNCF was naturally considered, as the
+  core focus of Cilium is to provide networking and security for cloud-native
+  environments and it goals align well with those of the Foundation.
+
+<!--2021-02-24-->
+* [_Sysdig contributes Falco's kernel module, eBPF probe, and libraries to the CNCF_](https://sysdig.com/blog/sysdig-contributes-falco-kernel-ebpf-cncf/).  
+  This contribution is a commitment provide and keep those components as open
+  source. The kernel module and eBPF probe component are roughly equivalent,
+  and implement a system call capture framework in the Linux kernel, used by
+  Sysdig and Falco. The libraries sit on top of them, and “enrich” the
+  information collected by the probes to turn it into meaningful data. All
+  components become the property of the Cloud Native Computing Foundation. Some
+  have been relicensed, although this does not seem to be the case of the eBPF
+  probes. From a technical point of view, this means that the contributors
+  worked on decoupling those from the Sysdig code base, making them easier to
+  reuse in other projects.
+
+  Nearly at the same time, the maintainers of the Falco submitted a proposal
+  for [CNCF graduation](https://github.com/cncf/toc/pull/641), which would
+  represent a marker of health and maturity for the project. And Sysdig, the
+  company behind Falco,
+  [raised a $188 million Series F](https://news.crunchbase.com/news/sysdigs-valuation-hits-1-19b-after-188m-series-f/)
+  at a $1.19 billion valuation.
+
+<!--2021-04-29-->
+* [_Microsoft acquires Kinvolk_](https://kinvolk.io/blog/2021/04/microsoft-acquires-kinvolk/).  
+  Launched in 2015, Kinvolk has been working on open source projects such as
+  the Flatcar Container Linux distribution, Lokomotive, or Inspektor Gadget.
+  The latter is a suite of eBPF-based tools for debugging and inspecting
+  Kubernetes clusters, and the company has been actively contributing to the
+  eBPF ecosystem (see for example the posts from Mauricio Vásquez Bernal
+  referenced later in this document). The company is joining the Azure team,
+  congratulations and best of luck for the future!
+
+<!--2021-04-06-->
+* [_LPC 2021 Networking and BPF Track CFP_](https://lore.kernel.org/bpf/6d225920-9ecc-ef24-2bf8-848ca86c7fb0@iogearbox.net),
+  <span style="white-space: nowrap;">on the development mailing list for eBPF.</span>  
+  Just like the previous editions, the next Linux Plumbers Conference will have
+  a _Networking and BPF_ track. The call for participation is open until the
+  **13th of August**. The conference itself is due to take place in Dublin,
+  Ireland, from the 27th to the 29th of September 2021.
```

Comment:
Might want to hold off on the dates here, there's been an update that the conference will go virtual and the dates may need to shift: https://www.linuxplumbersconf.org/blog/2021/index.php/2021/04/30/linux-plumbers-goes-fully-virtual/

Reply:
Well I assume the CFP won't change (cc @borkmann) but the conf itself might.

Reply:
Right, thank you! I changed to:
```markdown
  The call for participation is open until the
  **13th of August**. The conference itself was initially due to take place in
  Dublin, Ireland, from the 27th to the 29th of September 2021, but the
  organizers
  [recently announced](...)
  that it will be a fully virtual event instead.
```

Reply:
Yes, LPC is going to be virtual. We'll write that in our next CFP reminder.